Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1627 @@
+    """
+    Update payment status
+    """
+    # Validate IDs format
+    validate_uuid_format(payment_id, "payment_id")
+    validate_uuid_format(brand_id, "brand_id")
+    
+    try:
+        # Verify payment belongs to brand
+        payment_result = supabase.table("sponsorship_payments").select("*").eq("id", payment_id).eq("brand_id", brand_id).execute()
+        if not payment_result.data:
+            raise HTTPException(status_code=404, detail="Payment not found")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)